site/public/js/docker_interface.js

```diff
@@ -64,6 +64,40 @@ function addFieldOnChange() {
     }
 }
 
+function confirmationdialog(url, id) {
+    if (confirm(`Are you sure you want to remove ${id} image?`)) {
+        removeImage(url, id);
+    }
+}
+
+function removeImage(url, id) {
+    $.ajax({
+        url: url,
+        type: 'POST',
```

Comment:
You could use `$.post()` instead.

Reply:
I haven't seen $.post used in any of the code that I've worked on, and I ran a search and it seems to only be used twice in the entire codebase. Do you think this is a pressing change or can it wait for a later refactor that replaces a bunch of $.ajax requests with $.post?